translations/tiled_de.ts

```diff
@@ -3725,52 +3726,52 @@ Bitte Format angeben.</translation>
     <message>
         <location filename="../src/tiled/tilestampsdock.cpp" line="+196"/>
         <source>Delete Stamp</source>
-        <translation type="unfinished"></translation>
+        <translation>Stempel löschen</translation>
     </message>
     <message>
         <location line="+10"/>
         <source>Remove Variation</source>
-        <translation type="unfinished"></translation>
+        <translation>Variation entfernen</translation>
     </message>
     <message>
         <location line="+71"/>
         <source>Choose the Stamps Folder</source>
-        <translation type="unfinished"></translation>
+        <translation>Stempel-Ordner auswählen</translation>
     </message>
     <message>
         <location line="+15"/>
         <source>Tile Stamps</source>
-        <translation type="unfinished"></translation>
+        <translation>Kachel Stempel</translation>
```

Comment:
This sounds a bit weird to me though... In the context of the window only `Stamps` is used. But here it says `Tile Stamps`